src/components/FishCard.spec.js

```diff
@@ -23,7 +24,16 @@ describe('FishCard', () => {
     const fishName = screen.getByRole('heading', { level: 2 });
     expect(fishName).toHaveTextContent('Siamesischer Kampffisch');
 
+    const button = screen.getByRole('button', { name: /Weitere/i });
+    expect(button).toBeInTheDocument();
+
+    userEvent.click(button);
```

Comment:
When I am right, the userEvent.click is used in combination with a mock function in order to call it. Therefore both userEvents could be out here. However, you could test the toggle (if this is possible, im actually not sure  😅)

Reply:
My thought was that I test if the button is available and at the first click the list is displayed, then at the second click if the list is no longer displayed.

Reply:
well, lets say i am not the best one in testing😅
however, i think in that case you need to call the onClick={toggleProperties} with a mock function such as:
const toggleCallback = jest.fn();
nClick={toggleCallback} 
...
    expect(toggleCallback).toHaveBeenCalledWith({
      ...oneObject...
    });

Just an idea - might also not work 😊

Reply:
thanks for the hint, I'll take a look at it - testing is also not my big strength - as you can see :-D